src/sqlancer/mysql/gen/MySQLDeleteGenerator.java

```diff
@@ -49,8 +52,13 @@ private SQLQueryAdapter generate() {
                 "Truncated incorrect DOUBLE value" /*
                                                     * ignore as a workaround for https://bugs.mysql.com/bug.php?id=95997
                                                     */, "Truncated incorrect INTEGER value",
-                "Truncated incorrect DECIMAL value", "Data truncated for functional index"));
-        // TODO: support ORDER BY
+                "Truncated incorrect DECIMAL value", "Data truncated for functional index", "Unknown column"));
```

Comment:
We should add the expression errors as above, either if we generate a `WHERE` clause or `ORDER BY`. Like this, we won't need to duplicate the expected error here.